bcycle-backend/Data/BCycleContext.cs

```diff
@@ -20,31 +17,36 @@ public BCycleContext(DbContextOptions options) : base(options)
         public DbSet<Trip> Trips { get; set; }
         public DbSet<TripPhoto> TripPhotos { get; set; }
         public DbSet<TripPoint> TripPoints { get; set; }
-        
+        public DbSet<GroupTripCode> Codes { get; set; }
+
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
             modelBuilder.Entity<Trip>()
-                .HasMany(e => e.TripPoints)
-                .WithOne(tp => tp.Trip).HasForeignKey(tp => tp.TripId)
+                .HasMany(e => e.Route)
+                .WithOne(tp => tp.Trip)
+                .HasForeignKey(tp => tp.TripId)
                 .OnDelete(DeleteBehavior.Cascade);
 
             modelBuilder.Entity<Trip>()
-                .HasMany(e => e.TripPhotos)
-                .WithOne(tp => tp.Trip).HasForeignKey(tp => tp.TripId)
+                .HasMany(e => e.Photos)
+                .WithOne(tp => tp.Trip)
+                .HasForeignKey(tp => tp.TripId)
                 .OnDelete(DeleteBehavior.Cascade);
 
+            modelBuilder.Entity<GroupTripParticipant>()
+                .HasKey(p => new {p.UserId, p.GroupTripId});
+
             modelBuilder.Entity<GroupTrip>()
-                .HasMany(e => e.Participants)
-                .WithOne(p => p.GroupTrip).HasForeignKey(p => p.GroupTripId)
+                .HasMany(e => e.Route)
+                .WithOne(p => p.GroupTrip)
+                .HasForeignKey(p => p.GroupTripId)
                 .OnDelete(DeleteBehavior.Cascade);
 
-
-            modelBuilder.Entity<GroupTrip>().ToTable("GroupTrip");
```

Comment:
.ToTable() is required for the table names to match all the other documentation.

Reply:
No it is not. The tables are named with PascalCase by convention. You can check that up by inspecting migration file.

Reply:
Auto-generated names are plural, as you can see in the migration file:
![image](https://user-images.githubusercontent.com/48774838/79021534-0fd50000-7b7c-11ea-8ba3-b677504cb826.png)

Reply:
I missed it then. It is actually even better convention in my opinion. I'm more willing to change our database schema than this.